.coveragerc

```diff
@@ -107,6 +107,7 @@ omit =
     homeassistant/components/downloader.py
     homeassistant/components/feedreader.py
     homeassistant/components/garage_door/wink.py
+    homeassistant/components/garage_door/raspberry.py
```

Comment:
All Raspberry Pi components are prefixed with `rpi`. This one could be called `rpi_relay`